src/components/DrawableSVG/DrawableSVG.scss

```diff
@@ -0,0 +1,86 @@
+@use '../../styles/fonts' as *;
+@use '../../styles/variables' as *;
+@use '../../styles/mixins' as *;
+
+.svg__lock{
```

Comment:
svg classes share some code (e.g. display and position). Could we make this DRYer with an @extend?

Reply:
Good note, ended up changing to:
```
.drawable-svg{
  ...
  &--lock{
    ...
  }
}
```